### PR TITLE
Added Roman Numeral in Python Article

### PR DESCRIPTION
**Pull request by rayavarapuvikram1**
1. Added 2019-10-27-python.md to roman-numeral-conversion directory.
2. Modified a comment in code(line 43) and converted var= var + 4 to var +=4

Congrats on taking the first step to contributing to the Sample Programs repository maintained by [The Renegade Coder][1]! For simplicity, please make sure that your pull request includes one and only one contribution.

## Complete the Applicable Sections Below

Find which section best describes your pull request and make sure you fill it out. To start, let us know which issue you've fixed.

- [ ] I fixed #your-issue-number-here

### Code Snippets

- [ ] I named the pull request using `Added/Updated <Sample Program> in <Language>` format
- [ ] I created/updated the language README
  - [ ] I added the sample program name to the README
  - [ ] I added fun facts (i.e. debut, developer, typing, etc.)
  - [ ] I added reference link(s) to the README
  - [ ] I added solution citations when necessary (see [plagiarism][2])

### Documentation

- [ ] I named the pull request using `Added/Updated <Sample Program> in <Language> Article` format
- [ ] I followed the applicable article template
  - [ ] I added/updated a [language article][4] (i.e. The Python Programming Language)
  - [ ] I added/updated a [code snippet article][3] (i.e. Hello World in Perl)
  - [ ] I added/updated a [project article][5] (i.e. Fizz Buzz in Every Language)

### Testing

- [ ] I named the pull request using `Added/Updated <Language>/<Project> Testing` format
- [ ] I followed the testinfo template, if applicable

## Notes

Feel free to describe what you added or updated.

[1]: https://therenegadecoder.com/
[2]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/.github/CONTRIBUTING.md#plagiarism
[3]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/docs/templates/CODE_ARTICLE_TEMPLATE.md
[4]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/docs/templates/LANGUAGE_ARTICLE_TEMPLATE.md
[5]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/docs/templates/PROJECT_ARTICLE_TEMPLATE.md

### CODE REVIEW

File: docs/projects/roman-numeral-conversion/2019-10-27-python.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
+---
+title: Roman Numeral to Decimal number in Python
+layout: default
+last-modified: 2019-10-27
+featured-image: 
+tags: [python, roman_numeral_to_decimal_number]
+authors:
+  - auroq(code)
+  - rayavarapuvikram1(article)
```
Comment (jrg94) [inline docs/projects/roman-numeral-conversion/2019-10-27-python.md]: For this, you can just put yourself as the primary author. The parentheses will actually break our system.

```suggestion
  - rayavarapuvikram1
```

Don't forget to add yourself to the author file.

**Comment by rayavarapuvikram1** [comment]
Can it be committed now?

File: docs/projects/roman-numeral-conversion/2019-10-27-python.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
+---
+title: Roman Numeral to Decimal number in Python
+layout: default
+last-modified: 2019-10-27
+featured-image: 
+tags: [python, roman_numeral_to_decimal_number]
+authors:
+  - auroq(code)
```
Comment (jrg94) [inline docs/projects/roman-numeral-conversion/2019-10-27-python.md]:
```suggestion

```

File: docs/projects/roman-numeral-conversion/2019-10-27-python.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+  - rayavarapuvikram1
```
Comment (jrg94) [inline docs/projects/roman-numeral-conversion/2019-10-27-python.md]: Make sure you add yourself to https://github.com/TheRenegadeCoder/sample-programs/blob/master/docs/_data/authors.yml

Comment (rayavarapuvikram1) [reply docs/projects/roman-numeral-conversion/2019-10-27-python.md]: Done SIr.

**Pull request merged**